### PR TITLE
Improve read me

### PR DESCRIPTION
**Pull request by anasmi**
This change is [Reviewable](https://reviewable.io/reviews/vaadin/skeleton-starter-flow-spring/308)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,71 @@
...
-- [Using Vaadin and Spring](https://vaadin.com/docs/v14/flow/spring/tutorial-spring-basic.html) article
 
+If you want to run the application locally in the production mode, use `spring-boot:run -Pproduction` command instead.
+### Running Integration Tests
+
+Integration tests are implemented using TestBench. The tests take a few minutes to run and are therefore included in a separate Maven profile. To run the tests using Google Chrome, execute
+
+`mvn verify -Pit`
+
+and make sure you have a valid TestBench license installed. If the tests fail because of an old Chrome Driver or you want to use a different browser, you'll need to update the webdrivers.xml file in the project root.
```
Comment (samie) [inline README.md]: Would be good to have a link to TB, like:
[Vaadin TestBench](https://vaadin.com/testbench)

File: README.md [hunk 2]
```diff
@@ -1,24 +1,71 @@
...
+-Dcom.vaadin.testbench.Parameters.runLocally=chrome
+```
+
+If you would like to run a separate test make sure you have added these parameters to VM Options of JUnit run configuration
+
+## Project overview
+
+Project follow the Maven's [standard directory layout structure](https://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html):
+- Under the 'srs/main/java' are located Application sources
+   - `Application.java` is a starting point
```
Comment (samie) [inline README.md]: Runnable Java application class.

File: README.md [hunk 3]
```diff
@@ -1,24 +1,71 @@
...
+   - `GreetService.java` is a  Spring service class
+   - `MainView.java` is the class defining the entry point of the application
```
Comment (samie) [inline README.md]: Default view for the application. 

Or would it be even more familiar to talk about "page" here? Not sure.

File: README.md [hunk 4]
```diff
@@ -1,24 +1,71 @@
...
+- Under the 'srs\test' are located test files
+- `src\main\resources` contains configuration files and static resources
+- The `\frontend\` directory in the root folder contains client-side dependencies and resource files
+   - All CSS styles used by the application are located under the root directory 'frontend\styles'    
+   - Templates would be stored under the `\frontend\src`
+
+
+## More Information and Next Steps
+
+- Vaadin Basics [https://vaadin.com/flow](https://vaadin.com/flow)
```
Comment (samie) [inline README.md]: Perhaps instead https://vaadin.com/docs  ?

File: README.md [hunk 5]
```diff
@@ -1,24 +1,71 @@
...
+- Under the 'srs\test' are located test files
+- `src\main\resources` contains configuration files and static resources
+- The `\frontend\` directory in the root folder contains client-side dependencies and resource files
+   - All CSS styles used by the application are located under the root directory 'frontend\styles'    
+   - Templates would be stored under the `\frontend\src`
+
+
+## More Information and Next Steps
+
+- Vaadin Basics [https://vaadin.com/docs](https://vaadin.com/docs)
```
Comment (samie) [inline README.md]: Add link. "Download this and other examples from https://vaadin.com/start"

File: README.md [hunk 6]
```diff
@@ -1,24 +1,71 @@
...
+- More components at [https://vaadin.com/components](https://vaadin.com/components) and [https://vaadin.com/directory](https://vaadin.com/directory)
+- Using Vaadin and Spring :[Using Vaadin and Spring](https://vaadin.com/docs/v14/flow/spring/tutorial-spring-basic.html) article
```
Comment (samie) [inline README.md]: Extra colon and same text twice.

Comment (anasmi) [reply README.md]: Done

File: README.md (hunk 5 above)
Comment (anasmi) [reply README.md]: Done

**Pull request merged**